base/reflection.jl

```diff
@@ -2615,6 +2615,15 @@ function delete_method(m::Method)
     ccall(:jl_method_table_disable, Cvoid, (Any, Any), get_methodtable(m), m)
 end
 
+"""
+    seal_methodtable(m::Core.MethodTable)
```

Comment:
Perhaps the function name should end with a `!` since it modifies `mt`?

Also, if the flag this toggles is called `frozen`, would it make sense to have the associated verb be "freeze"? So e.g. this would be `freeze_methodtable!` (or just `freeze!`).

Reply:
If we change it to `freeze!`, what should we call the inverse, `thaw!`? :rofl:
So `seal/unseal` sounds better, what do you think?

Reply:
FWIW, `unfreeze` sounds fine to me.

Reply:
Agreed.

FFIW
https://github.com/JuliaLang/julia/issues/31222#issuecomment-469445675 also talked about sealed functions and sealed methods, which is related to this here.

But `freeze/unfreeze` is fine for me too.

Reply:
Renamed to `freeze!/unfreeze!`